cmd/kubeapps-apis/plugins/fluxv2/packages/v1alpha1/server.go

```diff
@@ -172,6 +201,130 @@ func (s *Server) GetAvailablePackages(ctx context.Context, request *corev1.GetAv
 	}, nil
 }
 
+// GetPackageMeta streams the package metadata based on the request.
+func (s *Server) GetPackageMeta(ctx context.Context, request *corev1.GetPackageMetaRequest) (*corev1.GetPackageMetaResponse, error) {
```

Comment:
I think this is what you were referring to, that I should ignore? As in, this will later be changed to `GetAvailablePackageDetail` or something? If so, why not just add it as we plan it to be (just this one call)?

Reply:
yeah, this will happen, don't worry about that